### DISCUSSION TITLE
How can I give some suggestions to GitHub about some ideas that I have?

### DISCUSSION THREAD

**Discussion by Rishabh-Singh-Codes** [community/community#43572] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have some suggestion regarding one of the features on GitHub, how can I submit it to them?

**Comment by auchynnikau** [comment, marked as answer]
You can create related ticket on [GitHub Support](https://support.github.com/request) portal.
